keyboards/dz60/keymaps/MintCollie/keymap.c

```diff
@@ -0,0 +1,51 @@
+#include QMK_KEYBOARD_H
+
+// Make special keycodes more visible
+#define ____ KC_TRNS
+#define XXXX KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

Also, because most, if not all internal keycodes do have a 7 character alias, this helps ensure that it's consistently spaced.